domain/workspace.go

```diff
@@ -26,8 +28,14 @@ type Workspace struct {
 	RecentlyVisitedAt   time.Time `json:"recentlyVisitedAt" db:"recently_visited_at"`
 }
 
+type CreateWorkspace struct {
+	Name    string `validate:"required"`
```

Comment:
Validate tag should be in payload only, please remove it

Reply:
Done. Removed validate tag in 6b7dc40